src/EindhovenLogo.jl

```diff
@@ -60,16 +60,287 @@ function draw_eindhoven_logo(;
     draw_vibes(scale=scale)
 end
 
-function draw_eindhoven(;scale::Real=4.5, textcolor="black")
+#= previous font-based version
+function draw_eindhoven(; scale::Real=4.5, textcolor="black")
     fontface("Bangla MN Bold")
-    fontsize(24.5*scale)
+    fontsize(24.5 * scale)
     sethue(textcolor)
-    textoutlines("eindhoven", O, :path, valign=:middle, halign=:center)
+    textoutlines("eindhoven", O, :path; valign=:middle, halign=:center)
     fillpreserve()
     #settext("eindhoven", Point(xoffset-215*scale, 110*scale))
     sethue(Luxor.julia_blue...)
-    circle(-45*scale, -7.5*scale, 2.5*scale, :fill)
+    return circle(-45 * scale, -7.5 * scale, 2.5 * scale, :fill)
 end
+=#
+
+function draw_eindhoven(; scale::Real=4.5, textcolor="black", dotcolor=Luxor.Colors.RGB(Luxor.julia_blue...))
```

Comment:
my goodness, how do you make these things?

Reply:
i have a little script that converts Swift graphics code to Cairo format and I have a MacOS app that copies graphics to the clipboard in Swift format … it’s occasionally necessary 😀